app/controllers/bootstrap.js

```diff
@@ -1,53 +1,44 @@
-myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout) {
-
-    $scope.Post = function($scope){
-      console.log('FIRED');
-       var data = {status:"Status of course",major:"the major of the course"};
-
-      $http.post('/api/post', data).then(function (successCallback, errorCallback){
-        console.log("successfilled");
-      });
-
+myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout, $rootScope, $location) {
+var parent = $rootScope;
+	$scope.status = {
+	    isFirstOpen: true,
+	    isFirstDisabled: false
+	  };
+    $scope.validateClick = function (group, index) {
+        if (group.correct == "addingDegreeCourse") {
+            group.isaddingDegreeCourse = true;
```

Comment:
When using camelCase ensure that you strictly adhere to camelCasing

group.is**A**ddingDegreeCourse